File: ozon-listing-auto/server/app/schemas/pace.py
"""节奏配置 API schema。"""
from pydantic import BaseModel

class PaceIn(BaseModel):
    min_interval_sec: int = 60
    max_interval_sec: int = 180
    daily_limit: int = 200
    active_hours: list[int] = [9, 23]
    wait_ozon_approval: bool = True

class PaceOut(PaceIn):
    task_id: int | None = None

File: ozon-listing-auto/web/src/api/pace.ts
import { api } from "./client";
export interface Pace { min_interval_sec: number; max_interval_sec: number; daily_limit: number; active_hours: number[]; wait_ozon_approval: boolean; }
export const getPace = (taskId: number) => api.get(`/pace?task_id=${taskId}`).then(r => r.data);
export const savePace = (taskId: number, body: Pace) => api.put(`/pace?task_id=${taskId}`, body).then(r => r.data);
